cogs/moderation/moderation.py

```diff
@@ -184,6 +183,51 @@ async def tempban(self, ctx: Context, member: discord.Member, length : CustomTim
 
             await ctx.send(embed=embed)
 
+    @commands.command(aliases=['purge', 'del', 'delete'])
+    @commands.guild_only()
+    @commands.has_permissions(manage_messages=True)
+    async def clear(self, ctx: Context, channel: Optional[discord.TextChannel], amount: int):
+        channel = channel or ctx.channel
+
+        await ctx.send(f"now purging {amount} messages...")
+
+        await channel.purge(limit=amount + 1)
+        self.bot.dispatch("purge", author=ctx.author)
+
+    @commands.command(name="unban")
+    @commands.guild_only()
+    @commands.has_permissions(ban_members=True)
+    async def _unban(self, ctx, id: int, *, reason="No Reason Provided"):
+        try:
+            user = discord.Object(id=id)
```

Comment:
```suggestion
            user = discord.Object(id=id)
```
you can just type hint to discord.Object here